src/client/opamPinCommand.mli

```diff
@@ -86,7 +86,3 @@ val parse_pins: string list -> (name * version option * url * string option) lis
     [url]), and returns it *)
 val read_opam_file_for_pinning:
   ?quiet:bool -> name -> OpamFile.OPAM.t OpamFile.t -> url -> OpamFile.OPAM.t option
-
-(** The default version for pinning a package: depends on the state, what is
-    installed and available, and defaults to [~dev]. *)
-val default_version: 'a switch_state -> name -> version
```

Comment:
For consistency, better put it on `OpamPinned`

Reply:
done

Reply:
Actually I can’t put in in `OpamPinned`:
```
  Error: dependency cycle between modules in _build/default/src/state:
     OpamUpdate
  -> OpamPinned
  -> OpamSwitchState
  -> OpamPinned
```

Reply:
hmm...